README.md

```diff
@@ -25,12 +25,30 @@ Example bean:
         </util:list>
     </bean>
 ```
+ And to use FastTokenServices with custom interval refresh(let us say One hour=3600,000L) of token keys, update spring config and specify an exact list of trusted issuers. 
```

Comment:
it may be easier to follow if you update the first example with TTL config, with a comment that it is optional along with the default value. Maybe also add a note in overview about caching of token keys.

Reply:
Also remove comma from 3600,000L.

Reply:
Definitely, but did this to preserve the current usage practice.